stdlib/sys.mli

```diff
@@ -208,6 +208,12 @@ external runtime_parameters : unit -> string = "caml_runtime_parameters"
     as the contents of the [OCAMLRUNPARAM] environment variable.
     @since 4.03 *)
 
+external poll_runtime : unit -> unit = "%poll"
+(** Run any pending actions from the runtime, such as signal
+    handlers, finalizers, or memprof callbacks. Any allocation
+    has this as a side-effect; call this to avoid such surplus
+    allocation. *)
```

Comment:
"; call this to avoid such surplus allocation" is cryptic to me. It reads as if calling this function would somehow reduce the number of allocations in the rest of the program. I would simply omit this sentence.